### PR TITLE
Image & input

### PR DESCRIPTION
**Pull request by amartya-rs**
feature update: image and input components

### CODE REVIEW

File: all-components/image.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+.res-img {
+    width: 100%;
+    height: auto;
+}
+
+.round-img {
+    object-fit: cover;
+    border-radius: 50%;
+}
```
Comment (DivyanshSareen) [inline all-components/image.css:12]: can use apsect-ratio: 1/1 to avoid elliptical image

**Review by DivyanshSareen — APPROVED** [review]
Clean and readable code. Good to merge.

File: all-components/image.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+.square-img {
+    object-fit: cover;
+    border-radius: 10px;
+}
+
```
Comment (zoef-yt) [inline all-components/image.css:18]: remove extra spacing

**Review by humzahasan — APPROVED** [review]
Good Selection of Class Name.  Code looks clean. Good to merge!

**Review by zoef-yt — APPROVED** [review]
Good to merge.

File: all-components/image.css (hunk 1 above)
Comment (amartya-rs) [reply all-components/image.css:12]: Will update, thanks for your suggestion.

File: all-components/image.css (hunk 2 above)
Comment (amartya-rs) [reply all-components/image.css:18]: Will take care of spacing.

**Pull request merged**